proc_linewidth.py

```diff
@@ -72,23 +86,32 @@
     print(type(result),result.shape)
     guess_nddata = nddata(result, [-1], ['$B_0$']).setaxis(
             '$B_0$',x_finer).set_units('$B_0$',d.get_units('$B_0$'))
-    fl.plot(d, label='data')
-    fl.plot(guess_nddata,':', label='guess')
-    fl.next('guess')
+    plot(d, label='data')
+    plot(guess_nddata,':', label='guess')
     model = s.Model({y_var:dVoigt})
-    guess = InteractiveGuess(model, y=d.data.real, B=d.getaxis('$B_0$'), n_points=500)
-    guess.execute()
-    print(guess)
+    if interactive:
+        guess = InteractiveGuess(model, y=d.data.real, B=d.getaxis('$B_0$'), n_points=500)
```

Comment:
what is this function InteractiveGuess? did we define it or is it some numpy thing?

Reply:
Symfit. You can tell this based on how it’s imported

Reply:
so is it a type of fit? similar to how global fit will fit a bunch of lines or how there are different minimizers/parameters for different fits?

Reply:
No -- this opens the window that allows you to adjust the parameters.  It's documented [here](https://symfit.readthedocs.io/en/stable/examples/ex_interactive_guesses_vector_2D.html)